src/main/web/src/apps/ontology/app-components/variable-details-view/variableDetails.html

```diff
@@ -74,7 +74,9 @@ <h2 class="om-panel-title underlined-heading">
 				{{'formField.property' | translate}}
 			</label>
 			<div class="col-sm-9">
-				<om-select name="omProperty" om-name="om-property" om-property="property" om-editing="editing" ng-model="model" om-options="data.properties" om-required="true"></om-select>
+				<om-select name="omProperty" om-name="om-property" om-property="property" om-editing="editing" ng-model="model" om-options="data.properties" om-required="true"
+					is-hyper-link="true" show-details="viewPropertyDetails(e)">
+				</om-select>
```

Comment:
Make sure editing of property/scale/method still works when it is opened through variable panel.